tests/unit_tests/simulator/test_simulation_context.py

```diff
@@ -59,3 +59,61 @@ def test_simulation_context(setup_case, storage):
         else:
             assert odd_ctx.job_status(iens) != JobStatus.FAILED
             assert odd_ctx.job_status(iens) == JobStatus.SUCCESS
+
+
+def test_simulation_context(setup_case, storage):
```

Comment:
Is this duplicating an exisiting test? Why did it have to be added

Reply:
It uses the other api, which in time everest will change over to. I will rewrite it to take the different APIs as pytest parameters instead